client/signup/steps/import-from/wix/index.tsx

```diff
@@ -1,21 +1,39 @@
 import { ProgressBar } from '@automattic/components';
-import { Progress, Title, SubTitle } from '@automattic/onboarding';
+import { Progress, Title, SubTitle, Hooray, NextButton } from '@automattic/onboarding';
 import { useI18n } from '@wordpress/react-i18n';
 import classnames from 'classnames';
 import React from 'react';
 
-const WixImporter: React.FunctionComponent = () => {
+import './style.scss';
+
+interface Props {
+	queryObject: {
+		temp: string;
+	};
+}
+const WixImporter: React.FunctionComponent< Props > = ( props ) => {
 	const { __ } = useI18n();
+	const { queryObject } = props;
 
 	return (
 		<div className={ classnames( 'import-layout__center' ) }>
-			<Progress>
-				<Title>{ __( 'Importing' ) }...</Title>
-				<ProgressBar color={ 'black' } compact={ true } value={ 33 } />
-				<SubTitle>
-					{ __( "This may take a few minutes. We'll notify you by email when it's done." ) }
-				</SubTitle>
-			</Progress>
+			{ queryObject.temp === 'progress' && (
+				<Progress>
+					<Title>{ __( 'Importing' ) }...</Title>
+					<ProgressBar color={ 'black' } compact={ true } value={ 33 } />
```

Comment:
Shouldn't `value` here also be a prop? Or are we planning to just wire this up with internal state?

Reply:
It is temporary stuff, the temporary way to show both screens we made. The next step is to include the business logic and I'll remove the temp query param support for sure.

Reply:
This is a temporary state; we built the screen and want to show how to consume it. The next phase is to include business logic.